backend/src/users/users.controller.ts

```diff
@@ -17,13 +18,15 @@ export class UsersController {
 
   @Get()
   @Roles(Role.ADMIN)
-  async findAll(@Query() query: FindAllQueryDto): Promise<IPaginationResult<User>> {
+  @UsePipes(new ValidationPipe({ whitelist: true, transform: true }))
```

Comment:
Don't we have global pipe for that?

Reply:
I didn't want to globally add the "whitelist true" field because I wasn't sure what impact it would have on the other endpoints, or how it would affect them.

Reply:
I think we can use whitelist flag on the global pipe 👍

Reply:
done